.github/workflows/code-style.yml

```diff
@@ -0,0 +1,31 @@
+name: Code Style
+
+on:
+  pull_request:
+
+jobs:
+  fix:
+    name: Fix Code Style
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2.1.1
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Fix Code Style
+        run: yarn fix
+
+      - name: Commit Changes
+        uses: stefanzweifel/git-auto-commit-action@v2.3.0
+        with:
+          commit_message: Apply Prettier changes
+          branch: ${{ github.head_ref }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Hope it will not create broken code due some bugs on our side or eslint :smile:

Reply:
Only run on pull request, don't worry.